en/docs/get-started/tutorials/create-your-first-integration.md

```diff
@@ -0,0 +1,134 @@
+# Create Your First Integration
+
+An integration is a component with logic to integrate APIs, microservices, applications, and data in different languages and formats.
+
+Consider a scenario where a developer has created an integration in [WSO2 Integration Studio](https://wso2.com/integration/integration-studio/) and wants to use it for API-led integration by exposing it in Choreo. In this tutorial, you will learn how to do the following actions to address this requirement:
```

Comment:
Does it change if the integration is created any other way? If not, shall we generalize this and say fo this tutorial we are using the integration studio based samepl?

Reply:
We assume(different IDEs can be used to develop from scratch.) MI integrations are developed only using developer studio. Hence development environment will be the developer studio always.

Reply:
Made no change here.